pages/linux/http-prompt.md

```diff
@@ -0,0 +1,15 @@
+# http-prompt
+
+> An interactive command-line HTTP client featuring autocomplete and syntax highlighting.
+
+- Start with the last session or http://localhost:8000:
```

Comment:
The description should aim at users starting with a clean slate. Perhaps - `Launch a session targeting the default url of http://localhost:8000 or the previous session` sounds better.

Reply:
Yeah, I used the examples from the official documentation.
Your description is much better. Thank you @agnivade